From 4b72e8cad5b9018335a06f380d4a6b7eee14da8d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Cerveau?= <scerveau@collabora.com>
Date: Sat, 1 Feb 2020 19:18:44 +0100
Subject: [PATCH] fdkaacdec: add support for mpegversion=2

Fix for #1199
---
 ext/fdkaac/gstfdkaacdec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/fdkaac/gstfdkaacdec.c b/ext/fdkaac/gstfdkaacdec.c
index ea5588a33..3345fcf0a 100644
--- a/ext/fdkaac/gstfdkaacdec.c
+++ b/ext/fdkaac/gstfdkaacdec.c
@@ -43,7 +43,7 @@ static GstStaticPadTemplate sink_template = GST_STATIC_PAD_TEMPLATE ("sink",
     GST_PAD_SINK,
     GST_PAD_ALWAYS,
     GST_STATIC_CAPS ("audio/mpeg, "
-        "mpegversion = (int) 4, "
+        "mpegversion = (int) {2, 4}, "
         "stream-format = (string) { adts, adif, raw }, " CHANNELS_CAPS_STR)
     );
 
-- 
2.24.1

